apps/testing-javascript/src/pages/playlists/index.tsx

```diff
@@ -18,103 +19,25 @@ const WorkshopsPage: React.FC<{modules: SanityDocument[]}> = ({modules}) => {
-                      <div className="pt-4 pb-3 font-mono text-xs font-semibold uppercase text-cyan-300">
-                        {state === 'draft' ? (
-                          <span className="mr-3 rounded-full bg-gray-700 px-2 py-0.5 font-sans font-semibold uppercase text-gray-200">
-                            Coming Soon
-                          </span>
-                        ) : (
-                          i === 0 && (
-                            <span className="mr-3 rounded-full bg-cyan-300 px-2 py-0.5 font-sans font-semibold uppercase text-black">
-                              New
-                            </span>
-                          )
-                        )}
-                        {sections && state !== 'draft' ? (
-                          <>
-                            {sections.length} sections,{' '}
-                            {sections.reduce(
-                              (acc: number, section: {lessons?: any[]}) =>
-                                section.lessons?.length
-                                  ? section.lessons?.length + acc
-                                  : acc,
-                              0,
-                            )}{' '}
-                            exercises
-                          </>
-                        ) : (
-                          <br />
-                        )}
-                      </div>
-
-                      {description && (
-                        <p className="text-gray-300">{description}</p>
-                      )}
-                      {state !== 'draft' && (
-                        <Link
-                          href={{
-                            pathname: '/playlists/[module]',
-                            query: {
-                              module: slug.current,
-                            },
-                          }}
-                          className="group mt-5 inline-block gap-2 rounded bg-gray-800 py-2 pl-4 pr-6 font-medium transition hover:bg-gray-700"
-                          onClick={() => {
-                            track('clicked view workshop module', {
-                              module: slug.current,
-                            })
-                          }}
-                        >
-                          <span className="pr-2">View</span>
-                          <span
-                            aria-hidden="true"
-                            className="absolute text-gray-300 transition group-hover:translate-x-1 group-hover:text-white"
-                          >
-                            →
-                          </span>
-                        </Link>
-                      )}
-                    </div>
-                  </li>
-                )
-              },
-            )}
+          <ul className="flex max-w-screen-lg flex-col gap-5 px-5 pt-10 sm:gap-8 sm:pt-20">
+            {modules.map((playlist) => {
+              return (
+                <Playlist
```

Comment:
Seems like we can reuse the `Playlist` component here instead of reworking and restyling a bunch of markup